src/scripts/gen-react-bindings.js

```diff
@@ -47,25 +47,30 @@ const getReactFileContents = async (svelteFilePath) => {
         ? `<${generics.map(g => g.name).join(', ')}>`
         : ''
 
-    const fileContents = `
-import type * as React from 'react';
-import SvelteToReact, { type ReactProps } from '${svelteReactWrapperRelativePath}';
-import ${componentName} from './${fileName}';
-import type { ${componentName}Events as SvelteEvents, ${componentName}Props as SvelteProps } from './${fileName}';
+    const binding = `
```

Comment:
I split this into a separate declaration & bindings file because it makes the build process simpler, and it means we don't have to do the monster `as unknown as CrazyType` we were doing before.